TACC-Cloud/hazmapper
# task/WG-227 add v3 auth support

## Overview: ##

Add v3 auth support

## PR Status: ##


## Related Jira tickets: ##

* [WG-227](https://tacc-main.atlassian.net/browse/WG-227)

## Summary of Changes: ##

## Testing Steps: ##
1. Log in and see map project listings
2. Use `featue/tapisv3` branch on geoapi
